src/main/java/org/spdx/sbom/gradle/utils/SpdxDocumentBuilder.java

```diff
@@ -161,6 +161,8 @@ public SpdxDocumentBuilder(
 
     this.resolvedArtifacts =
         resolvedArtifacts.entrySet().stream()
+            .filter(
+                e -> !(e.getKey().getComponentIdentifier() instanceof ProjectComponentIdentifier))
```

Comment:
I guess that technically this means that `this.resolvedArtifacts` has a slightly different meaning than `resolvedArtifacts` - should we rename `this.resolvedArtifacts` to something like `this.resolvedExternalArtifacts`?

Reply:
yeah makes sense, lemme rename